### DISCUSSION TITLE
/ignore to work across Bungee

### DISCUSSION THREAD

**Discussion by Verdux** [Zrips/CMI#2625] [Feature Requests]
Add /ignore support for CMI Bungee so players don't see public/private messages from players on their ignore list across multiple servers

### Basic explanation as to why this feature is needed and/or useful to have: ###

Currently players can ignore a player on Server 1 but still see their messages when they go to Server 2. Makes it a pain to need to ignore them on ever server linked. Would be nice to just ignore them once and it will recognise it across the network

**Comment by Insprill** [comment]
yes that would be cool. but also have an ability in the config to enable/ disable it, or enable it between say server 1 and 2 but not 3
